rllib/algorithms/ppo/ppo_learner.py

```diff
@@ -241,7 +241,7 @@ def additional_update_for_module(
     @OverrideToImplementCustomLogic
     def _compute_values(
         self,
-        batch_for_vf: Union[MultiAgentBatch, NestedDict],
+        batch_for_vf: Dict[str, Any],
```

Comment:
I see we swtich now to the plain dictionary. I remember the `ActorCriticEncoder` uses indeed a two level `NestedDict` - is this also planned for these classes?

Reply:
Not yet, but yes, I would like to simplify things into using pure dicts.
This will be a separate PR, in which I'll try for the new API stack to not depend on these classes anymore: SampleBatch, MultiAgentBatch, NestedDict.